Sources/Core/Sources/ECS/Singles/InputState.swift

```diff
@@ -24,6 +24,14 @@ public final class InputState: SingleComponent {
   /// The position of the right thumbstick.
   public private(set) var rightThumbstick: Vec2f = Vec2f(0, 0)
 
+  /// The time since the last time the player pressed the forwards key.
+  public private(set) var forwardsDownTime: Int = 0
+    /// Whether the spring was triggerf by a double tap
```

Comment:
Typos on this line as well

Reply:
There's still two typos lol; too much indentation and "spring".
No need to create an extra commit to resolve this, just combine it with stackotter's requested changes